File: getglyphogram.py
from bs4 import BeautifulSoup;
import requests

url = "http://www.signbank.org/signpuddle2.0/translate.php";

r = requests.post(url+"?ui=1&sgn=4&build=building&action=translate")

xmltext = r.text
xml = xmltext.encode('utf-8')

soup = BeautifulSoup(xml, "lxml")
imgs = soup.find_all("img")

print ("<html><body>");
for img in imgs:
    src = img["src"]
    if src.startswith("glyphogram.php"):
        src = 'http://www.signbank.org/signpuddle2.0/'+src
        img['src'] = src
        print(img)
print ("</body></html>");

File: getglyphogram.js
// nodejs
var JSSoup = require('jssoup').default;

var url = "http://www.signbank.org/signpuddle2.0/translate.php";

// i don;t have a bot
var data;
function querySW() {
$.post(url+"?ui=1&sgn=4&build=building&action=translate", null, function(data, status, xhr) {
	console.log ("<html><body>");
	console.log(data);
	var xmltext = r.text
	var xml = xmltext.encode('utf-8')

	var soup = new JSSoup(xml)
	var imgs = soup.findAll("img")

	for (img in imgs) {
	    var src = imgs[img]["src"];
	    if (src.indexOf("glyphogram.php") > 1) {
		src = 'http://www.signbank.org/signpuddle2.0/'+src;
		imgs[img]['src'] = src;
		console.log(imgs[img]);
	     }
	}
	console.log ("</body></html>");
 });
};
